### PR TITLE
flake8.ymlの作成

### PR DESCRIPTION
**Pull request by sizumita**

### CODE REVIEW

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        pip install flake8
+        flake8 . --count --show-source --statistics
```
Comment (aiotter) [inline .github/workflows/flake8.yml:27]: Installing flake8 with pip can be an independent job. Doing this enables you to know an error is occurred either on installing flake8 or linting the code.

**Comment by aiotter** [comment]
You may refer to this page: https://docs.github.com/ja/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template

File: .github/workflows/flake8.yml (hunk 1 above)
Comment (aiotter) [reply .github/workflows/flake8.yml:27]: You may refer to this page: https://docs.github.com/ja/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template

**Comment by sizumita** [comment]
hmmm I see

File: .github/workflows/flake8.yml (hunk 1 above)
Comment (sizumita) [reply .github/workflows/flake8.yml]: resolved

**Comment by sizumita** [comment]
まーじします

**Pull request merged**